src/device.rs

```diff
@@ -349,14 +397,28 @@ pub enum DeviceMode {
 mod tests {
     use super::*;
 
+    #[test]
+    fn test_core_range_ordering() {
+        let all = CoreRange::All;
+        let core0 = CoreRange::Range((0, 0));
+        let core0_1 = CoreRange::Range((0, 1));
+        let core0_3 = CoreRange::Range((0, 3));
+        let core2_3 = CoreRange::Range((2, 3));
+
+        assert!(all < core0);
+        assert!(core0 < core0_1);
+        assert!(core0_1 < core2_3);
+        assert!(core2_3 < core0_3);
```

Comment:
`core1`에 대한 케이스도 있으면 의도가 더 쉽게 읽힐 것 같습니다. 어떨까요?
(all이 최우선, 코어 수가 적을수록 우선, 같으면 낮은 인덱스가 우선)

```suggestion
        let all = CoreRange::All;
        let core0 = CoreRange::Range((0, 0));
        let core1 = CoreRange::Range((1, 1));
        let core0_1 = CoreRange::Range((0, 1));
        let core0_3 = CoreRange::Range((0, 3));
        let core2_3 = CoreRange::Range((2, 3));

        assert!(all < core0);
        assert!(core0 < core1);
        assert!(core1 < core0_1);
        assert!(core0_1 < core2_3);
        assert!(core2_3 < core0_3);
```

Reply:
네, 더 명확한것 같습니다! :)